python-bindings/module.cpp

```diff
@@ -77,6 +42,51 @@ PYBIND11_MODULE(pyTTD, m) {
             }
     );
 
+    py::class_<TTD::TTD_Replay_ActiveThreadInfo, std::unique_ptr<TTD::TTD_Replay_ActiveThreadInfo, py::nodelete>>(m, "ActiveThreadInfo")
+        .def_property("threadid", [](TTD::TTD_Replay_ActiveThreadInfo& self) {
+            return self.info->threadid;
+        }, nullptr)
+        .def_property("next_major", [](TTD::TTD_Replay_ActiveThreadInfo& self) {
+            return self.next.Major;
+        }, nullptr)
+        .def_property("next_minor", [](TTD::TTD_Replay_ActiveThreadInfo& self) {
+            return self.next.Minor;
+        }, nullptr)
+        .def_property("last_major", [](TTD::TTD_Replay_ActiveThreadInfo& self) {
+            return self.last.Major;
+        }, nullptr)
+        .def_property("last_minor", [](TTD::TTD_Replay_ActiveThreadInfo& self) {
+            return self.last.Minor;
+        }, nullptr);
+
+    py::class_<CONTEXT, std::unique_ptr<CONTEXT, py::nodelete>>(m, "Context")
+        .def_readonly("SegCs", &CONTEXT::SegCs)
```

Comment:
*cosmetics* I rather prefer (and found it more Pythonic) to have attribute in snake_case, starting with a lower letter

Reply:
OK I will rewrite this part (totally agree with the remark)